libs/ui/src/election_info_bar.tsx

```diff
@@ -37,49 +38,45 @@ export function ElectionInfoBar({
   machineId,
   precinctSelection,
 }: Props): JSX.Element {
-  const electionDate = format.localeDate(
-    new Date(electionDefinition.election.date)
-  );
+  const {
+    election: { precincts, date, title, county, state, seal, sealUrl },
+  } = electionDefinition;
+  const electionDate = format.localeDate(new Date(date));
 
   /* istanbul ignore next */
   const theme: Theme = { ...(contrastTheme.dark ?? {}) };
 
   return (
     <Bar theme={theme} data-testid="electionInfoBar">
+      {(seal || sealUrl) && <Seal seal={seal} sealUrl={sealUrl} />}
```

Comment:
Not in this PR, but I think we should just do away with non `sealUrl` seals because of the added complexity.

Reply:
I'm on the fence. 

## Inline Seal
- Full election definition portability
- Can use our products without first having to commit your seal to our repo.

## Seal URL
- There is simplicity with only referring to seals which are added to the codebase.
- Only seals in the codebase can be used.
- Assets have to be added to every app (currently, though perhaps a build process could solve this?)